### PR TITLE
Clarify axis coordination for offset measurements

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: website/docs/apps/filter_app/README.md [hunk 1]
```diff
@@ -76,10 +76,31 @@ its operational environment and physical characteristics.
...
-and should originate from the central axis of the robot.
-Please follow the indicated measurement guide to maintain consistency and reliability in data collection.
+:::info INFO
+**Amiga Calibration Guide**
+
+To ensure accurate calibration of your Amiga robot, please adhere to the following measurement guidelines:
+
+1. **Unit of Measurement**: All measurements must be taken in meters (m).
+
+2. **Reference Point**: Begin all measurements from the central axis of the robot.
```
Comment (Hackerman342) [inline website/docs/apps/filter_app/README.md:86]: Suggestion to clarify what central axis means. I.e., center of the robot (in length & width) at ground level.

**Review by Hackerman342 — APPROVED** [review]
Good clarification

**Review by edgarriba — COMMENTED** [review]
a figure would be more illustrative

**Comment by Hackerman342** [comment]
> a figure would be more illustrative

Agreed, but we need to change what we are asking users to measure (see https://github.com/farm-ng/workspace/issues/2115) to make it less error prone. I would hold off on spending the time to make a figure until we've made that change in the app.

cc @edgarriba @guilhermedemouraa

File: website/docs/apps/filter_app/README.md (hunk 1 above)
Comment (guilhermedemouraa) [reply website/docs/apps/filter_app/README.md]: Clarified here 0ef4d0bf

**Pull request merged**